Cargo.toml

```diff
@@ -6,5 +6,7 @@ members = [
     "tokio-trace-tower-http",
     "tokio-trace-log",
     "tokio-trace-env-logger",
-    "tokio-trace-slog"
+    "tokio-trace-slog",
+    "tokio-trace-macros/tokio-trace-macros",
+    "tokio-trace-macros/example"
```

Comment:
This change to the package structure is kind of weird. 

If you look at the other crates, I believe you'll see that we have an `examples` dir inside of the dir with the crate's name, at the same level as the source dir. For example, the `tokio-trace` crate currently looks like this:

```
tokio-trace
├── Cargo.toml
├── examples
│   ├── basic.rs
│   └── sloggish
│       ├── main.rs
│       └── sloggish_subscriber.rs
└── src
    ├── dedup.rs
    ├── dispatcher.rs
    ├── instrument.rs
    ├── lib.rs
    ├── span.rs
    └── subscriber.rs
```

Is it possible to structure the macros crate the same way, or is there some proc-macro-specific issue preventing this?

Reply:
> It looks like removing this feature will allow this branch to build successfully on the latest stable compiler (1.30). Can we get rid of it?

Yep!

> Is it possible to structure the macros crate the same way, or is there some proc-macro-specific issue preventing this?

Maybe—the primary requirement is that proc-macros have to be in their own crate.

Reply:
D'oh, I was wrong. Apparently proc macros don't need that silly organization fixed.